### ISSUE TITLE
Filter events

### ISSUE THREAD

**Issue by RolfKoenders** [gitpoint/git-point#116]
Adding a filter on the Events/Feed screen can be helpful to only show what you want to see. 
Possible filters: 
- Comments
- Stars on repo's
- Pull Request actions
- Issue actions

So you can for example easily see which repos got starred.

**Comment by housseindjirdeh** [comment]
Yes this would be nice to have, how are you envisioning we allow filtering? A button group up top just like we have in the `Notifications` screen?

There are [a lot](https://developer.github.com/v3/activity/events/types/) of possible events so I like the list you currently have:

* All (the default)
* Stars
* Comments
* Pull Requests (includes all Pull Request activity including comments)
* Issues (includes all Issue activity including comments)

Will love to hear everybody's suggestions

**Comment by RolfKoenders** [comment]
Yeah those are the events we want to filter on most likely. We can always add a option later on. 
I was actually more thinking like a filter icon/button on the top right of the Events navigation bar. A button group will be coming really cluttered with more then 3 a 4 options. 
Will look into more possible options.

**Comment by housseindjirdeh** [comment]
Agreed an icon might make more sense actually, cheers thank you :raised_hands:
